test/e2e/volume_provisioning.go

```diff
@@ -350,14 +358,27 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 	})
 })
 
-func verifyDefaultStorageClass(c clientset.Interface, expectedDefault bool) {
-	sc, err := c.StorageV1().StorageClasses().Get("default", metav1.GetOptions{})
+func getDefaultStorageClassName(c clientset.Interface) (name string, found bool, err error) {
```

Comment:
I reworked getDefaultStorageClassName to be extra paranoid and do `framework.Failf` on any error